mind_matter_api/api/surveys.py

```
@@ -17,7 +17,9 @@ def init_survey_routes(app):
 
     @app.route('/surveys', methods=['GET'])
     def get_surveys():
-        """Retrieve all surveys"""
+        user_id = get_authenticated_user_id_or_abort()
+        if isinstance(user_id, tuple): return user_id
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)